Update banner-notifications.mdx (#2363)
grommet/hpe-design-system: aries-site/src/pages/templates/global-banner-notifications.mdx

<Example
  showResponsiveControls={['fullScreen']}
  // height, width, and screenContainer.scale value chosen to best visually 
  // represent the example
  height="650px"
  width="100%"
  caption="An example of a dashboard with a Global Banner warning the user that their subscription is expiring. The user can either dismiss the notification or click on the anchor to renew their subscription."
  screenContainer={{ scale: 0.57 }}
>
  <BannerContentLayoutExample />
</Example>

## Guidance
Global Banners are high-attention notifications used primarily for status updates on background processes and upcoming system changes and releases.

### Usage
- Global Banners are used to showcase Warning and Critical statuses or informational alerts. Depending on the severity of the information, they may remain persistent across multiple sessions or be dismissible by the user.
- Since Global Banners take over the top of an interface and can be intrusive, they should be reserved only for important system generated updates.
- Also when applicable, Global Banners should always have a relevant anchor(s) to direct users to further actions or to view more about the issue in full detail.

> ### Do not use Global Banners for:

Reviewer: ```suggestion
### Do Not Use Global Banners For
```

> - Completion of tasks, such as long running or batch actions.

Reviewer: It seems like all of these use cases are for "Toast Notifications"? But it seems like we are only saying the last bullet point is. Maybe we separate the sub-bullet into it's own paragraph? see suggestion below.

- Success or failure confirmations.
- Low-attention and low-severity information.

>   - For low-severity notifications that do not require user action or user attention such as confirmation receipts, instead use [Toast Notifications](templates/toast-notifications).

Reviewer: ```suggestion

For low-severity notifications that do not require user action or user attention such as confirmation receipts, instead use [Toast Notifications](templates/toast-notifications).
```

Reviewer 2: @taysea good suggestion. Also, did you want me to commit these suggestions or leave that to you and Brittany?

Reviewer: If you're good with the copy suggestion, go for it! @KennyAtHPE

### Things to Avoid

> - Do not use use close icon when information is important and needs to remain persistent. Also consider whether or not the information should remain persistent across multiple sessions

Reviewer: ```suggestion
- Do not use `onClose` when information is important and needs to remain persistent. Also, consider whether or not the information should remain persistent across multiple sessions.
```
also, I don't understand the second sentence. are we saying that if information needs to remain persistent across multiple sessions it should never be dismissible? or that we should persist a banner until a user dismisses it (meaning it could persist over multiple sessions but they can dismiss it anytime, afterwhich it will not appear?) @KennyAtHPE

Reviewer 2: It'd be the latter, maybe something along the lines of "Also, global banners may remain persistent across multiple sessions until they are dismissed." would be more clear.

Reviewer: Cool then I think we should move that sentence from the "Things to Avoid" and instead just place it under guidance. @britt6612

> - Global Banners are not clickable, instead use anchors to direct users to take further action or to view the issue in full detail

Reviewer: ```suggestion
- Global Banners are not clickable. Instead use anchors to direct users to take further action or to view the issue in full detail.
```

> - Overly long notifications can take up lots of screen real estate on smaller screens

Reviewer: ```suggestion
- Do not allow messages to exceed X characters.
```
Feels like we should be specific about a character count. What does "overly long" mean? seems subjective. @KennyAtHPE

Reviewer 2: I think we should use the same limitations we imposed before which is the 192 character limit

## Anatomy
<Example
  caption="Diagram illustrating the four parts composing of a Banner Notification."
  plain
>
  <BannerNotificationDiagram />
</Example>

> 1. Status Indicator is an icon made of a shape and color to capture the user’s attention

Reviewer: ```suggestion
1. **Status Indicator** is an icon made of a shape and color to capture the user’s attention
```

> 2. Message is a short description explaining the issue

Reviewer: ```suggestion
2. **Message** is a short description explaining the issue.
```

> 3. Anchors (optional)  are used to direct users to links related to the notification

Reviewer: ```suggestion
3. **Anchors (optional)** are used to direct users to links related to the notification.
```

> 4. Close Button (optional) is used to dismiss the notification

Reviewer: ```suggestion
4. **Close Button (optional)** is used to dismiss the notification.
```

> ## Best Practices 

Reviewer: ```suggestion
## Best Practices

```
